### PR TITLE
simplify index updating, optimize for heroku cron

### PR DESCRIPTION
**Pull request by indirect**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -34,57 +34,27 @@ rescue LoadError => e
...
-  if res.response['Location']
-    modified?(res.response['Location'], cache_file)
-  elsif res.is_a?(Net::HTTPSuccess)
-    File.open(cache_file, 'w') {|file| file.write res.body }
-    true
-  else
-    false
+    Zlib::GzipReader.open(file) do |gz|
+      Marshal.load(gz)
+    end
```
Comment (andremedeiros) [inline Rakefile]: Sounds to me like we could be rescuing and retrying a couple times here maybe?

Comment (indirect) [inline Rakefile]: We already run this task as a corn every 60 minutes, so retrying didn't seem super important to me.

**Comment by indirect** [comment]
@homu r+

**Comment by indirect** [comment]
...hmm, guess I should add this repo to homu.

**Pull request merged**